packages/manager/apps/ips/src/routes/routes.constant.ts

```diff
@@ -25,7 +25,7 @@ export const urls = {
   listing: subRoutes.root,
   listingTerminate: `${subRoutes.root}/${subRoutes.terminate}/${urlDynamicParts.id}`,
   listingUpsertDescription: `${subRoutes.root}/${subRoutes.upsertDescription}/${urlDynamicParts.id}`,
-  listingConfigureGameFirewall: `${subRoutes.root}/${urlDynamicParts.id}/${subRoutes.configureGameFirewall}`,
+  configureGameFirewall: `${subRoutes.root}/${urlDynamicParts.id}/${subRoutes.configureGameFirewall}`,
```

Comment:
I preferred listingConfigureGameFirewall as it make more sense as we have this action in listing page knw ? What do you think ?